packages/replayer/src/replay.utils.ts

```diff
@@ -92,6 +97,12 @@ export const createReplayPage: (options: {
   );
   await page.evaluateOnNewDocument(userInteractionsFile);
 
+  // If virtual time is enabled then we need to install the virtual event loop before
+  // any customer code runs, since that code may snapshot copies of the setTimeout etc. functions
```

Comment:
I think I would avoid leaving comments like this in the open

Reply:
Chatted on Slack to understand reasoning: reasoning was that comment adds bloat & not required.